src/style.css

```diff
@@ -1,3 +1,67 @@
 body {
   margin: 0;
+  padding: 0;
+}
+
+html {
+  font-size: 12px;
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+}
+
+.header {
+  background-color: #fff;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 50px;
+}
+
+.nav__list {
+  display: flex;
+  margin: 0;
+  list-style: none;
+  padding: 0;
+  align-items: center;
+}
+
+.nav__item {
+  margin-left: 20px;
+}
+
+.nav__item:first-of-type {
+  margin: 0;
+}
+
+.nav__link {
+  display: block;
+  color: #000;
+  text-decoration: none;
+  height: 60px;
+  line-height: 60px;
+  text-transform: uppercase;
+}
+
+.is-active {
+  color: #00acdc;
+}
+
+.nav__link:hover {
+  color: #00acdc;
+}
```

Comment:
CSS Variable is used for a blue color - add this one as well
<img width="493" alt="image" src="https://github.com/user-attachments/assets/d59b78ba-38fe-4e62-b1ff-320c13566f8a">